source/manual/create-a-local-transaction.html.md

```diff
@@ -0,0 +1,236 @@
+
+```bash
+bundle exec rails service:enable[LGSL_CODE,LGIL_CODE,SERVICE_NAME,SERVICE_SLUG]
+```
+
+Then run [this rake task](https://github.com/alphagov/local-links-manager/blob/master/lib/tasks/import/missing_links.rake) in Local Links Manager to add (empty) links for all available local authorities...
+
+```bash
+bundle exec rails import:missing_links
+```
+
+## Upload Local Authority URLs to Local Links Manager
+
+__Note__: These steps must be complete before starting:
+
+* [Get details of or create a service](#get-details-of-or-create-a-service)
+* [Get a list of local authority slugs and URLs](#get-a-list-of-local-authority-slugs-and-urls)
+* [Add the service to Local Links Manager](#add-the-service-to-local-links-manager)
+
+Log into Backend and upload the completed CSV file...
+
+```bash
+$ gds govuk connect ssh -e integration backend
+
+# If you run this rake task in Jenkins, you'll need run scp-push against all
+# backend machines. Otherwise you can omit `:MACHINE_NO`.
+$ gds govuk connect scp-push -e integration backend:MACHINE_NO LOCAL_FILE_PATH REMOTE_FILE_PATH
+```
+
+Run [this rake task](https://github.com/alphagov/local-links-manager/blob/master/lib/tasks/import/service_links_from_csv.rake) to import the links from the CSV file.
+
+```bash
+bundle exec rails import:service_links[LGSL_CODE,LGIL_CODE,REMOTE_FILE_PATH]
+```
+
+## Add content and publish the service in Publisher
+
+Once __all__ of the above steps are complete, ask a member of your content design team to add any desired content and publish the service in [Publisher](https://publisher.publishing.service.gov.uk). The content design team will need to know the [LGSL code](#lgsl-code), the [LGIL code](#lgil-code), and that the page needs to be created as a new `Local transaction` format in publisher.
+
+## Glossary
+
+### LGSL code
+
+A Local Government Services List (LGSL) is a number that LLM uses to uniquely identify a service. They originate in ESD where it is used as the service ID.
+
+### LGIL code
+
+Along with services, ESD has the concept of an Interaction. Functionally, the relationship between service and interaction is many-to-many.
+
+An interaction describes the reason, use, or purpose of the service.
+
+Like an [LGSL code](#lgsl-code), LLM uses the interaction ID from ESD to uniquely identify an interaction.  LLM this the Local Government Interaction List (LGIL) code.
+
+One thing to note is that ESD has deprecated the use of interactions. LLM has not however been updated to accommodate the replacement concept in ESD. This means that services in LLM still need a relationship with one-or-more interaction in order to function correctly. If you are unsure which interaction(s) it is that you need, it is recommended that the Providing information (LGIL code 8) interaction be used as this is the generic in nature.
+
+### Seek content design advice
```

Comment:
This is only referred to in one place, so perhaps this content should be moved into that section?

Reply:
Actually, this one is referenced in a few places:

```
* A [SERVICE_NAME](#seek-content-design-advice)
* A [SERVICE_DESCRIPTION](#seek-content-design-advice)
* The [SERVICE_SLUG](#seek-content-design-advice)
```

```
| [LGSL_CODE](#lgsl-code) | [SERVICE_DESCRIPTION](#seek-content-design-advice) | [PROVIDING_TIER](#providing-tier) |
```

and 

```
This can be configured with or without `button_text` and `button_link`. [Seek content design advice](#seek-content-design-advice) with the content.
```

So I'll won't change this. Thanks